src/datepicker/datepicker.ts

```diff
@@ -139,6 +149,12 @@ export class NgbDatepicker implements OnChanges,
    */
   @Input() startDate: {year: number, month: number};
 
+  /**
+   * An event fired when navigation happens and currently displayed month changes.
+   * The payload is the new month we're navigating to.
+   */
+  @Output() navigate = new EventEmitter<{year: number, month: number}>();
```

Comment:
`preventDefault()` could potentially make sense as well :-) So yeh, maybe let's do an event with `current` and `next` to keep our options open?